user-interface/src/data-verification/ConsolidationCasesTable.tsx

```diff
@@ -82,7 +88,26 @@ function _ConsolidationCaseTable(
                 </td>
                 <td scope="row">{formatDate(bCase.dateFiled)}</td>
                 <td scope="row" className="text-no-wrap">
-                  {''}
+                  {!props.isAssignmentLoaded && (
+                    <LoadingSpinner
+                      id={`loading-spinner-case-asignment-${bCase.caseId}`}
+                      height="1rem"
+                      caption="Loading..."
+                    />
+                  )}
+                  {props.isAssignmentLoaded &&
+                    bCase.attorneyAssignments &&
+                    bCase.attorneyAssignments.length > 0 && (
+                      <ul className="usa-list--unstyled">
+                        {bCase.attorneyAssignments.map((att, idx) => (
+                          <li key={idx}>{att.name}</li>
```

Comment:
Is `idx` unique enough to use as the key here?

Reply:
Probably not.